Neos.Neos/Resources/Private/Translations/en/Modules.xlf

```diff
@@ -428,6 +428,12 @@
 			<trans-unit id="users.tooltip.edit" xml:space="preserve">
 				<source>Edit user</source>
 			</trans-unit>
+			<trans-unit id="users.tooltip.editDisallowed" xml:space="preserve">
```

Comment:
If I don't get it wrong, the entries for alle the new flash messages are missing.

Reply:
Sorry, I don't get what you mean. I added them for English, must I add them for every language?

Reply:
No, not for every language, but you need to register the sentence with its id for english and use this id in the flashMessage, like it is done here for example:

- https://github.com/neos/neos-development-collection/blob/4d5219277da26e62f6ce5335d74a1ddd1ac1a86d/Neos.Media.Browser/Classes/Controller/TagController.php#L75
- https://github.com/neos/neos-development-collection/blob/7584a1d6ef3666c471a5ae9147dd585c6cf77a7c/Neos.Media.Browser/Resources/Private/Translations/en/Main.xlf#L400

Sadly, it seems that a lot of flash messages in the user module are not translated yet. But I would like to not add more untranslated messages.

Reply:
AFAIS the items added are in the XLIFF file, everything else was already there (and if it was missing in XLIFF already, that is nothing that should block this one…